xls/passes/rematerialization_pass.cc

```diff
@@ -157,11 +165,12 @@ absl::flat_hash_map<Node*, Slack> ComputeSlack(
         }
       }
 
+      // longest path from current node to any stage exit node
       int64_t node_to_exit_max = 0;
-      for (Node* entry : entry_nodes) {
-        if (longest.at(entry).contains(node)) {
```

Comment:
compute `entry_to_node_max` again

Reply:
Ah, woops, looks like that was a copy error.